### PR TITLE
feat: small stylistic improvements

### PR DESCRIPTION
**Pull request by flekschas-ozette**
This PR introduces some smaller stylistic improvements to ensure that `cev` is usable in dark mode

### Light Mode

Screenshot 2023-05-18 at 11 09 18 AM

### Dark Mode

Screenshot 2023-05-18 at 11 08 53 AM

### CODE REVIEW

File: src/cev/_embedding_comparison_widget.py [hunk 1]
```diff
@@ -20,15 +20,15 @@
...
+from cev.components import MarkerSelectionIndicator, WidthOptimizer
 
 
 def _create_titles(
     titles: tuple[str, str]
 ) -> tuple[ipywidgets.Widget, ipywidgets.Widget]:
     left_title, right_title = titles
     spacer = ipywidgets.HTML(
-        value='<div style="height: 1px; background: #efefef;" />',
+        value='<div style="height: 1px; background: var(--jp-border-color0);" />',
```
Comment (manzt) [inline src/cev/_embedding_comparison_widget.py:31]: Might have an issue with this outside of JupyterLab, but I think it's fine for now.

File: src/cev/components/_width_optimizer.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+class WidthOptimizer(anywidget.AnyWidget):
+    _esm = """
+    export function render(view) {
+      setTimeout(() => {
+        view.el.parentNode.style.setProperty('--jp-widgets-inline-label-width', 'auto');
+      }, 0);
+    }
+    """
```
Comment (manzt) [inline src/cev/components/_width_optimizer.py:13]: can you write a comment explaining what this is doing?

File: src/cev/_embedding_comparison_widget.py (hunk 1 above)
Comment (flekschas-ozette) [reply src/cev/_embedding_comparison_widget.py:31]: Yeah, for now I consider this widget to be Jupyter only

File: src/cev/components/_width_optimizer.py (hunk 2 above)
Comment (flekschas-ozette) [reply src/cev/components/_width_optimizer.py:13]: Oh sure, sorry I forgot. Usually the ipywidgets have this odd whitespace before the label starts:

Screenshot 2023-05-18 at 1 35 34 PM

That whitespace is defined by `--jp-widgets-inline-label-width` on the labels. By unsetting the variable the whitespace is removed. (E.g., see screenshots in the PR description)

**Review by manzt — APPROVED** [review]
I like the changes. Just wondering if now we are getting pretty coupled to JupyterLab. Not a huge deal at the moment.

**Comment by flekschas-ozette** [comment]
I think for now the coupling is fine. When we get closer to v1 we can generalize the styling

**Comment by manzt** [comment]
Sounds good, feel free to merge when ready 🎸

File: src/cev/components/_width_optimizer.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import anywidget
+
+__all__ = ["WidthOptimizer"]
+
+
+class WidthOptimizer(anywidget.AnyWidget):
```
Comment (flekschas-ozette) [inline src/cev/components/_width_optimizer.py:6]:
```suggestion
class WidthOptimizer(anywidget.AnyWidget):
    """This widget gets rid of unwanted whitespace in front of ipywidgets"""
```

**Pull request merged**